test/languages/common-lisp_test.rb

```diff
@@ -0,0 +1,36 @@
+require "test_helper"
+
+module SnippetExtractor
+  module Languages
+    class CommonLispTest < Minitest::Test
+      def test_full_example
+        code = <<~CODE
+          ;;;
+          ;;; Example of how to convert aribic numerals to roman numerals
+          ;;;
+          (defpackage #:roman-numerals
```

Comment:
For example the `defpackage` expression is basically noise and I would like to skip it.